src/utils/errorUtils.ts

```diff
@@ -0,0 +1,7 @@
+import {Exception, Options} from "../ExceptionTransformerModel";
+
+function getErrorDetail(errorInfo: Exception | null | undefined){
```

Comment:
Otherwise, it can return `false | null | undefined | ExceptionDetail`. We can simplify what it returns

```suggestion
function getErrorDetail(
  errorInfo: Exception | null | undefined
): ExceptionDetail | null {
  return errorInfo && typeof errorInfo.detail === "object"
    ? errorInfo.detail
    : null;
}
```